### DISCUSSION TITLE
Cannot buy premium

### DISCUSSION THREAD

**Discussion by DragonTayl** [aclap-dev/video-downloadhelper#572] [Help and Q&A]
### Your question

I am on the VDH site and attempting to purchase the premium version but the "buy" button does not do anything.

### Addon & CoApp version and details

Version 9.0.1.13 (stable), target google, locale en-US. Companion App installed (v2.0.19)

**Comment by jcfrog** [comment]
hi,
could it be a region problem? May I ask where you are?

**Comment by DragonTayl** [comment]
Sure!

I'm in coastal North Carolina, USA

The free version worked for downloading, with the companion app.

**Reply by jcfrog** [reply]
So it's not a region problem... :/

**Comment by paulrouget** [comment]
You confirm you're using Chrome ?

**Comment by DragonTayl** [comment]
Chrome is up to date
Version 125.0.6422.77 (Official Build) (64-bit)

Copy/paste,

Thank you for helping!
